README.md

```diff
@@ -15,6 +15,7 @@ It builds upon Flink's `TestEnvironment` and `TestStreamEnvironment` and can be
 
 | Version |  Flink Version | Java Version |
 | --------------- | ------------- | ------------ |
+| 0.5    | 1.6.3  | 1.8 |
```

Comment:
Is there any reason this is not compatible with version 1.6.0?

Reply:
No, it will also be compatible with 1.6.0.